charts/kong/example-values/dbless-config.yaml

```diff
@@ -0,0 +1,9 @@
+_format_version: "1.1"
+services:
+  # Example configuration
+  - name: example.com
+    url: http://example.com
+    routes:
+    - name: example
+      paths:
+      - "/example"
```

Comment:
Per the other comment, just remove this, since we use this directory for other content and don't need to provide anything different than upstream docs.

Reply:
[done](https://github.com/Kong/charts/pull/465/commits/071a59910032ba6435c86197ce0a93ab5b800a9f)